src/main/java/frc/robot/commands/boathookCommands/setpointCommands/Angle1BoathookCommand.java

```diff
@@ -24,9 +24,9 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    boathook.setAngle(boathook.getLevel().angle1);
+    boathook.setAngle(boathook.getLevel().angle1 + boathook.microRotationOffset);
     System.out.println("CURRENT ANGLE: " + boathook.getAngle());
-    System.out.println("SET ANGLE: " + boathook.getLevel().angle1);
+    System.out.println("SET ANGLE: " + (boathook.getLevel().angle1 + boathook.microRotationOffset));
```

Comment:
For logging, we should probably make these be smart dashboard things or on the Teleoperated tab or something. They're very noisy in the logs otherwise since they run every 20ms.